setup.py

```diff
@@ -6,7 +6,7 @@
     author_email="cristian@cs.cornell.edu",
     url="https://github.com/CornellNLP/ConvoKit",
     description="ConvoKit",
-    version="2.5.3",
+    version="2.6.0",
```

Comment:
Probably best to remove this before merging if we're sure we're not doing a 2.6 release.